pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -47,28 +48,13 @@ let
     };
   };
 
-  # Derived from subprojects/implot.wrap
-  implot = rec {
-    version = "0.16";
-    src = fetchFromGitHub {
-      owner = "epezent";
-      repo = "implot";
-      rev = "refs/tags/v${version}";
-      hash = "sha256-/wkVsgz3wiUVZBCgRl2iDD6GWb+AoHN+u0aeqHHgem0=";
-    };
-    patch = fetchurl {
-      url = "https://wrapdb.mesonbuild.com/v2/implot_${version}-1/get_patch";
-      hash = "sha256-HGsUYgZqVFL6UMHaHdR/7YQfKCMpcsgtd48pYpNlaMc=";
-    };
-  };
-
   # Derived from subprojects/imgui.wrap
   imgui = rec {
     version = "1.89.9";
     src = fetchFromGitHub {
       owner = "ocornut";
       repo = "imgui";
-      rev = "refs/tags/v${version}";
+      tag = "v${version}";
```

Comment:
Not so big fan that changes like this flowed into the formatting commit.

Reply:
Oh yep sorry! Squashed it into the wrong commit.